python/lsst/ts/standardscripts/base_track_target.py

```diff
@@ -96,14 +96,20 @@ def get_schema(cls):
                 properties:
                   ra:
                     description: ICRS right ascension (hour).
-                    type: number
-                    minimum: 0
-                    maximum: 24
+                    anyOf:
+                      - type: number
+                        minimum: 0
+                        maximum: 24
+                      - type: string
+                        pattern: "^[+-]?[0-9]{1,2}:[0-9]{1,2}:[0-9]*"
```

Comment:
No [+-] needed for ra, correct?

Reply:
Right, just the positive sign no negative.